### PR TITLE
web: add `What's included` section into root README.md for web

### PR DESCRIPTION
**Pull request by eymar**

### CODE REVIEW

File: tutorials/Web/README.md [hunk 1]
```diff
@@ -1,12 +1,103 @@
...
 
 [Handing Events](Events_Handling/README.md) - a short overview of Events handling with compose web
 
 [Style DSL](Style_Dsl/README.md) - about styling the composable components in web
+
+## What's included:
+
+### Module `compose.runtime`
+It provides fundamental building blocks of Compose's programming model and state management.
+`Compose for Web` uses Jetpack's runtime implementation.
```
Comment (olonho) [inline tutorials/Web/README.md:21]: Compose's

Comment (eymar) [reply tutorials/Web/README.md:21]: @olonho Do you mean there is a typo?
I'm not sure I understand where it is.

The description for `runtime` was verified with: https://developer.android.com/jetpack/androidx/releases/compose-runtime#kts

Comment (eymar) [reply tutorials/Web/README.md:21]: or do you want to clarify the last sentence?

`Compose for Web` uses Jetpack's runtime implementation. -> `Compose for Web` uses Compose Jetpack's runtime implementation.

Comment (olonho) [reply tutorials/Web/README.md:21]: Jetpack is set of Android libraries, using it in Web tutorial is misleading and incorrect.

Comment (eymar) [reply tutorials/Web/README.md]: fixed

**Comment by eymar** [comment]
Merging now, but please let me know if anything can be improved

**Pull request merged**